<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3840</width>
    <height>2160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>syncMRT</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1440</width>
     <height>900</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="2">
     <widget class="QStackedWidget" name="workStack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>100</height>
       </size>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QWidget" name="widgetSidebarList" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QWidget" name="widgetSidebarStack" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>225</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>225</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="3">
     <widget class="QWidget" name="variableWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="toolbarPane">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>31</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="open">
     <property name="title">
      <string>Open...</string>
     </property>
     <addaction name="menuFileOpenXray"/>
     <addaction name="menuFileOpenCT"/>
     <addaction name="menuFileOpenMRI"/>
     <addaction name="menuFileOpenRTP"/>
     <addaction name="separator"/>
     <addaction name="menuFolderOpen"/>
    </widget>
    <addaction name="open"/>
    <addaction name="separator"/>
    <addaction name="m_fe"/>
   </widget>
   <widget class="QMenu" name="tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="m_txr"/>
    <addaction name="m_ta"/>
   </widget>
   <addaction name="file"/>
   <addaction name="tools"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <action name="menuFileOpenXray">
   <property name="text">
    <string>X-ray</string>
   </property>
  </action>
  <action name="menuFileOpenCT">
   <property name="text">
    <string>CT</string>
   </property>
  </action>
  <action name="menuFileOpenMRI">
   <property name="text">
    <string>MRI</string>
   </property>
  </action>
  <action name="m_txr">
   <property name="text">
    <string>Capture X-ray</string>
   </property>
  </action>
  <action name="menuFileOpenRTP">
   <property name="text">
    <string>Treatment Plan</string>
   </property>
  </action>
  <action name="m_ta">
   <property name="text">
    <string>Align</string>
   </property>
  </action>
  <action name="m_fe">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menuFolderOpen">
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
